python/GafferSceneTest/SceneAlgoTest.py

```diff
@@ -301,19 +301,31 @@ def testHistory( self ) :
 
 			self.assertIsNone( history )
 
-			# Before running the same test again, set up an expression that pulls an unrelated piece of a
-			# scene plug, before running this test again.  This isn't part of attribute history,  so it
-			# shouldn't affect the result
+		runTest()
 
-			attributes["sourceScene"] = GafferScene.Sphere()
-			attributes["boundQuery"] = GafferScene.BoundQuery()
-			attributes["boundQuery"]["scene"].setInput( attributes["sourceScene"]["out"] )
-			attributes["boundQuery"]["location"].setValue( "/sphere" )
+		# Before running the same test again, set up an expression that pulls an unrelated piece of a
+		# scene plug, before running this test again.  This isn't part of attribute history,  so it
+		# shouldn't affect the result
 
-			attributes["expression"] = Gaffer.Expression()
-			attributes["expression"].setExpression(
-				'b = parent["boundQuery"]["bound"]; parent["attributes"]["NameValuePlug"]["value"] = b.size()[0]'
-			)
+		attributes["sourceScene"] = GafferScene.Sphere()
+		attributes["boundQuery"] = GafferScene.BoundQuery()
```

Comment:
This is pre-existing code, but reading it again I was baffled about why we're parenting these nodes under the StandardAttributes node. I think they can just be local variables instead? Oh, wait, is this some scheme to give the expression a `parent` scope to operate in? Maybe we could avoid all that by just feeding the output from the bound query into a V3f attribute?